### ISSUE TITLE
Not possible to use dynamic variables in if statements

### ISSUE THREAD

**Issue by janhsch** [sawhney17/logseq-smartblocks#30]
Hi! I really appreciate your work; it's excellent!

I would like to use your plugin to be able to create daily-sensitive journal templates. More specifically, I have a "stuff for tomorrow" section in my journal template where I would like to reference <%tomorrow%> as the entries in this section would be shown in the reference list of tomorrow's journal. And it's just working fine if I enter "Stuff for <% tomorrow %>. 

However, as I use logseq only for work purposes, it would be awesome to see "Stuff for <% next monday %>" in the journal of a given friday instead of "Stuff for <%tomorrow%>, as this would be a saturday. I tried to solve this issue by using if statements (my week starts at monday (= 0)):

`Stuff for <% if dayofweek = 4 || if dayofweek = 5: <%Next Monday%> %><% if dayofweek = 0 || if dayofweek = 1 || if dayofweek = 2 || if dayofweek = 3 || if dayofweek = 6: <%Tomorrow%> %>`

Unfortunately, the result I'm getting today (i.e., a Friday) is `Stuff for  <%Next Monday %> `%>`` and on any workday, I'm getting `Stuff for  <%Tomorrow %> %>`, which means that the dynamic variables in the if statement won't get resolved.

Is there any possibility of fixing this? I would really appreciate your help!

**Comment by sawhney17** [comment]
So do you mean something like maybe “<%next workday%>”?

**Comment by janhsch** [comment]
Yes, that would be just awesome and would do the trick to 100%! As this is no supported dynamic variable, I'm searching for a workaround for quite a while.
